searchcore/src/vespa/searchcore/proton/server/visibilityhandler.cpp

```diff
@@ -50,23 +50,23 @@ void VisibilityHandler::commitAndWait()
     _writeService.summary().sync();
 }
 
-bool VisibilityHandler::startCommit(const LockGuard & unused)
+bool VisibilityHandler::startCommit(const LockGuard &unused, bool force)
 {
     (void) unused;
     SerialNum current = _serial.getSerialNum();
-    if (current > _lastCommitSerialNum) {
+    if (current > _lastCommitSerialNum || force) {
```

Comment:
I prefer parentheses so you do not need complete operator precedence ruleset control...